_posts/2020-11-05-welcome-to-jekyll-demo-post.markdown

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title:  "Bioinformatic files: types and formats"
+date:   2020-11-06 12:54:04 -0700
+categories: blogpost
+---
+# Introduction
+In bioinformatics there are many different file types to store information from sequencing and -omics work. The most common file types are generally FASTA and FASTQ. But there are a lot of other semi-common file types as well like SAM, BAM, VCF as well as countless other self made file types for individual purposes.
+
+In this post I will cover four file types that are covered in the BTEC 640 course. The first two are the most common which are FASTA and FASTQ. The second two are slightly less common but still worked with frequently which are SAM and VCF.
+
+# FASTA
+
+FASTA stands for FAST-All. It was built off an earlier FASTP format that was designed around protein sequences and then expanded to encompass any kind of sequence format. Hence why the descriptor "All".  
+```fastq
```

Comment:
Same comment as below. Jekyll doesn't understand sequencing formats, so you should remove `fastq` from after the backticks. And also it's not fastq anyway...